src/welcome-day-1.md

```diff
@@ -18,10 +18,10 @@ Please remind the students that:
 * They should ask questions when they get them, don't save them to the end.
 * The class is meant to be interactive and discussions are very much encouraged!
   * As an instructor, you should try to keep the discussions relevant, i.e.,
-    keep the related to how Rust does things vs some other language. It can be
+    keep it related to how Rust does things vs some other language. It can be
     hard to find the right balance, but err on the side of allowing discussions
     since they engage people much more than one-way communication.
-* The questions will likely mean that we about things ahead of the slides.
+* The questions will likely mean that we are about to see things ahead in the slides.
```

Comment:
I think I wanted to say "The questions will likely mean that we talk about things ahead of the slides". I'm basically trying to tell the instructor

1. Expect many questions
2. Embrace this: make the class interactive and engaging (fun!) for the students
3. Don't worry about repeating yourself — repetition is good when you try to teach something.

When I've been teaching the class, I find myself saying "You've already seen _X_ yesterday" a lot... and then I can go a bit quicker through the slides since I can see that the class already got the point of _X_.

Reply:
I think the bullet points nicely convey your thoughts on their own.  May I suggest replacing this sentence with your bullet points?